examples/transmitter/swagger_server/test/test_stream_management_controller.py

```diff
@@ -461,7 +494,7 @@ def test_verification_request__pushing(client: FlaskClient, new_stream: Stream,
             headers={'Authorization': f'Bearer {new_stream.client_id}'}
         )
 
-    assert response.status_code == 204, "Incorrect response code: {}".format(response.status_code)
+    assert response.status_code == 204, "Incorrect response code: {}".format(response.status_code) + response.data.decode('utf-8')
```

Comment:
This line is painfully long. Can you create the message on the line above like so (preferring f-strings to `format()`)
```
err_msg = (
    f"Incorrect response code: {response.status_code} "
    f"{response.data.decode('utf-8')}"
)
assert response.status_code == 204, err_msg
```

Reply:
I replaced all these asserts with a call to `assert_status_code` function.